.github/workflows/mobile-traffic_director.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     permissions:
       contents: read
       packages: read
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
cc @abeyad

Reply:
This workflow is no longer needed, and has been disabled for some time now, so I sent a PR to remove it: https://github.com/envoyproxy/envoy/pull/38429